### DISCUSSION TITLE
Merging Various Tests Run into One

### DISCUSSION THREAD

**Discussion by asmarazaq** [test-results-reporter/reporter#213] [Q&A] [answered]
My test suite runs in 5 containers and the test results are showed in Teams' alerts in 5 different shards. Is there a way to merge all those shards into one at Teams' channel?

**Comment by ASaiAnudeep** [comment, marked as answer]
We can use wildcards to read multiple result files at once and send it as a single report.
https://docs.testbeats.com/guides/results.html#wildcards
